script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -261,71 +267,72 @@ But for now we should hurry to school. You're not gonna be late on my watch."
 [sha:1b81dc6965bbf2d7452031c5eba3ef198b3ba10b]{
 -- Page 53, Offset 14931.
 -- 先輩は校舎に歩いていく。
-Senpai starts walking toward school.
+Senpai starts walking toward the school.
 }
 [sha:85a9e7aaab7af282af82c49cc327b49e64a95f3c]{
 -- Page 53, Offset 14932.
 -- いつの間にか正門には誰もいない。先輩を問い詰めている間に、ホームルームまであと３分という状況になっていた。
-By now, there are no students left by the school entrance. As I was hurling question to Senpai, time passed to the point that now only 3 minutes remain until the start of homeroom.
+All the other students that had been milling around by the entrance disappeared while I wasn't paying attention.
+ I spent so long trying to get an answer out of Senpai that there's now only three minutes left until homeroom.
 }
 [sha:93bb88db7099ca1716b7cddb81aab0205a70834b]{
 -- Page 54, Offset 14933.
 -- 文句もあるけど今は先輩の言う通りだ。
 // I added the latter sentence, maybe it's not needed.
-I still have things I want to say to her, but she's right: I have to go to class.
+I'm still not done with her, but she's right――I have to get to class.
 }
 [sha:9904d2924cc5eb600e9068ad9c20c717c255f264]{
 -- Page 54, Offset 14934.
 -- まだ動転している心臓を落ち着かせながら、先輩の後に付いていった。
-I try to calm my palpitating heart as I follow Senpai toward the building.
+Trying to calm my racing heart, I follow after Senpai.
 }
 [sha:7533c56d731d9fd7e38b02635b72114e2e2d8215]{
 -- Page 55, Offset 14935.
 -- ホームルームまであと１分。
-There's 1 minute left until homeroom starts.
+There's only one minute left before homeroom starts.
 }
 [sha:974e285f4570ef3b8cc7a2008fab4cdb3b918d3f]{
 -- Page 55, Offset 14936.
 -- 廊下に生徒の姿はない。
-There are no students in the corridors.
+The corridor is completely devoid of students.
 }
 [sha:cdb0167e400b00b0cd6723a84d3d172865a2b7c9]{
 -- Page 55, Offset 14937.
 -- あるのは遅刻間際で小走りになる俺と先輩の足音と、
 // found this hard to phrase; pay special attention in editing
-The only thing there is the sound of footsteps from the two people almost late for class.
+The only things present in it are the footsteps of two people that are on the verge of being late for class, and―――
 }
 [sha:be3ade05ea3b04e55e983566b634e889ae6e8789]{
 -- Page 56, Offset 14938.
 -- 「あ、きたきた～♪　志貴クン、おはよー！」
-"Oh, arrived at last ♪
- Mornin' Shiki-kun!"
+"Oh, here he comes ♪%{n}
```

Comment:
maybe extend the "comes" to "cooomes" to make it more sing-songy? Hard to emphasize the second half of the word in text but it reads a little dry as-is.